src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -37,6 +37,10 @@ public string ZipUrl
                 {
                     return Environment[EnvironmentSettingNames.AzureWebsiteZipDeployment];
                 }
```

Comment:
Do we also need to update the logic here https://github.com/Azure/azure-functions-host/blob/06bfd89b780f8810bee11f9a843a46f96048a1d9/src/WebJobs.Script/Environment/EnvironmentExtensions.cs#L63?